tests/test_ruleset.py

```diff
@@ -0,0 +1,102 @@
+from xpotato.dataset.utils import default_pn_to_graph
+from xpotato.graph_extractor.extract import FeatureEvaluator
+from xpotato.graph_extractor.rule import RuleSet, Rule
+
+import os
+
+dir_name = os.path.dirname(os.path.realpath(__file__))
+
+FEATURE1 = [["(u_0 / find :obl (u_1 / .*) :obj (u_2 / .*))"], [], "FIND"]
+FEATURE2 = [["(u_0 / write :nsubj (u_1 / .*) :obj (u_2 / .*))"], [], "WRITE"]
+
+FEATURE3 = [
+    ["(u_0 / find :obl (u_1 / .*) :obj (u_2 / .*))"],
+    [],
+    "FIND",
+    [{"ARG1": 1, "ARG2": 2}],
```

Comment:
What are these?

Reply:
@Eszti New arguments for OpenIE, you can find examples in https://github.com/adaamko/POTATO/blob/dev/notebooks/openie.ipynb